### PR TITLE
Biome

### PR DESCRIPTION
**Pull request by sachaw**
@ajmcquilkin Happy to merge now if you want, we can work on any further linting as commits to master, this PR has removed all of the old dependencies & there config in favour of Biome.js, CI has also been updated to reflect this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,6 @@
     "rust:test": "cd src-tauri; cargo test",
     "ui:dev": "vite",
     "ui:build": "tsc && vite build",
-    "ui:lint": "eslint src/*.{ts,tsx}",
```
Comment (ajmcquilkin) [inline package.json:13]: Can we expose a command for devs to manually lint the project?

**Comment by ajmcquilkin** [comment]
Looks good other than the missing command! Not sure why the lint CI and Windows testing is failing, I'll dig in if I get a sec

File: package.json (hunk 1 above)
Comment (sachaw) [reply package.json:13]: Fixed

**Comment by sachaw** [comment]
Tests seem to all be passing, we can slowly fix all of the new lints as we go.

**Pull request merged**